AUTHORS

```diff
@@ -1,4 +1,4 @@
-Main author: Jörg Müller <nexyon@gmail.com>
```

Comment:
I cannot see the difference between the two fields here, maybe it's because there is a BOM added or removed? Please make sure it's consistent throughout the repository, I also saw files below having the issue.